### ISSUE TITLE
Scheduler focused in-place pod resize test is flaky and attempts negative resource values

### ISSUE THREAD

**Issue by vinaykul** [kubernetes/kubernetes#116415]
### Which jobs are failing?

ci-cos-cgroupv2-inplace-pod-resize-containerd-main-e2e-gce

### Which tests are failing?

Kubernetes e2e suite: [It] [sig-node] [Serial] Pod InPlace Resize Container (scheduler-focused) [Feature:InPlacePodVerticalScaling] pod-resize-scheduler-tests 

### Since when has it been failing?

March 08, 2023

### Testgrid link

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-cos-cgroupv2-inplace-pod-resize-containerd-main-e2e-gce/1633599468962058240

### Reason for failure (if possible)

Kubernetes e2e suite: [It] [sig-node] [Serial] Pod InPlace Resize Container (scheduler-focused) [Feature:InPlacePodVerticalScaling] pod-resize-scheduler-tests expand_less | 1s
-- | --
{ failed [FAILED] Error creating Pod: Pod "testpod1" is invalid: [spec.containers[0].resources.limits[cpu]: Invalid value: "-57m": must be greater than or equal to 0, spec.containers[0].resources.requests[cpu]: Invalid value: "-57m": must be greater than or equal to 0] In [It] at: test/e2e/framework/pod/pod_client.go:98 @ 03/08/23 22:52:29.043 }

### Relevant SIG(s)

/sig node

**Comment by SergeyKanzhelev** [comment]
@vinaykul is it mostly green now? Is this bug still active?

**Comment by vinaykul** [comment]
It is green because we serialized the tests. If it's ok, I'd like to keep this open for tracking purposes when we add a job that runs as many tests in parallel as possible, we investigate the reason for this failure if we can repro it and make the test airtight.
/assign

**Comment by mmiranda96** [comment]
/remove-lifecycle rotten
/remove-lifecycle stale

@vinaykul should we keep this issue open? It seems that after serializing the tests everything went green.

**Comment by tallclair** [comment]
Here's the new testgrid link, and this is consistently green: https://testgrid.k8s.io/sig-node-containerd#cos-cgroupv1-inplace-pod-resize-containerd-e2e-serial

It looks like the cgroupv2 version isn't running for some reason though.

I think we should close this, and open a new issue as needed. I suspect the underlying flakiness would be addressed by some of the feedback I left on https://github.com/kubernetes/kubernetes/pull/124296

/close
